homeassistant/components/atag/strings.json

```diff
@@ -6,14 +6,14 @@
         "title": "Connect to the device",
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
-          "email": "Email (Optional)",
+          "email": "[%key:common::config_flow::data::email%] (Optional)",
```

Comment:
You have to replace the full key, not partially

Reply:
Would it be advisable to remove the (Optional) part.
Or should I create a new entry in the base strings.json with something called email_optional?

Reply:
Nah, just skip the optional part